code/game/objects/items/tools/crowbar.dm

```diff
@@ -70,16 +70,44 @@
 	usesound = 'sound/items/jaws_pry.ogg'
 	force = 15
 	toolspeed = 0.25
-	var/airlock_open_time = 100 // Time required to open powered airlocks
+
+obj/item/crowbar/power/examine()
```

Comment:
Missed a `/` on the start of this

Also I think you should put `mob/user` in the args of this, to remain consistent with the rest of the codebase